### PR TITLE
[Magiclysm] Adding more attunement spells (2/3)

### PR DESCRIPTION
**Pull request by GuardianDll**
#### Summary
Mods "Adding more attunement spells"
#### Purpose of change
Adding:
Magnetism mage "Electrolaser" spell (high damage electric attack)
Ice elemental "Cold heart" spell (cheap middle damage slow down attack)
Cleansing flame "Hermes reborn" spell (healing and pain split spell with small damage for not 100% healing)
#### Testing
~~I didnt test it yet, cause i did it a long time ago, but ill test if in few hours~~ tested, works fine
#### Additional context
Some of the changes that was maded by maintainers were lost, like some descriptions errors (sorry for that), so i hope someone can check them and some stuff that was added after content freeze that i didnt notice.
~~Also its not linted yet.~~ linted

### CODE REVIEW

**Comment by actual-nh** [comment]
I'm not a maintainer, BTW.

File: data/mods/Magiclysm/Spells/attunements/Magnetism_Mage.json [hunk 1]
```diff
@@ -217,5 +217,33 @@
     "min_field_intensity": 1,
     "max_field_intensity": 1,
     "field_chance": 2
+  },
+  {
+    "id": "magnetismmage_electrolaser",
+    "type": "SPELL",
+    "name": "Electrolaser",
+    "description": "Creates a conductive channel from the caster's hand to the target by ionizing air molecules using a high-powered laser beam, then sends a powerful half million volt charge down the channel.",
```
Comment (actual-nh) [inline data/mods/Magiclysm/Spells/attunements/Magnetism_Mage.json:225]:
```suggestion
    "description": "Creates a conductive channel from the caster's hand to the target by ionizing air molecules using a high-powered laser beam, then sends a powerful half-million volt charge down the channel.",
```

File: data/mods/Magiclysm/Spells/attunements/Cleansing_Flame.json [hunk 2]
```diff
@@ -75,5 +75,44 @@
...
+    "effect": "attack",
+    "shape": "blast",
+    "base_casting_time": 350,
+    "base_energy_cost": 800,
+    "energy_source": "MANA",
+    "difficulty": 10,
+    "min_damage": -20,
+    "max_damage": -120,
+    "damage_increment": -2.5,
+    "max_level": 40,
```
Comment (KorGgenT) [inline data/mods/Magiclysm/Spells/attunements/Cleansing_Flame.json:95]:
```suggestion
    "max_level": 35,
```

File: data/mods/Magiclysm/Spells/attunements/Cleansing_Flame.json [hunk 3]
```diff
@@ -75,5 +75,44 @@
...
+    "extra_effects": [ { "id": "cleansingflame_hermes_burn" }, { "id": "pain_split" } ],
+    "flags": [ "SOMATIC", "VERBAL", "CONCENTRATE", "LOUD" ]
+  },
+  {
+    "id": "cleansingflame_hermes_burn",
```
Comment (KorGgenT) [inline data/mods/Magiclysm/Spells/attunements/Cleansing_Flame.json:100]: this spell doesn't have a max level

File: data/mods/Magiclysm/Spells/attunements/Ice_Elemental.json [hunk 4]
```diff
@@ -22,5 +22,48 @@
...
+    "effect": "attack",
+    "shape": "blast",
+    "damage_type": "cold",
+    "min_damage": 20,
+    "max_damage": 80,
+    "damage_increment": 1.5,
+    "min_range": 6,
+    "max_range": 20,
+    "range_increment": 0.35,
+    "max_level": 40,
```
Comment (KorGgenT) [inline data/mods/Magiclysm/Spells/attunements/Ice_Elemental.json:42]:
```suggestion
    "max_level": 35,
```

File: data/mods/Magiclysm/Spells/attunements/Ice_Elemental.json [hunk 5]
```diff
@@ -22,5 +22,48 @@
...
+  {
+    "id": "cold_heart_slow",
+    "type": "SPELL",
+    "name": "Cold Heart slowdown",
+    "description": "Causes slowdown from the Cold Heart spell.  Subspell; you aren't allowed to see it except in debug mode.",
+    "valid_targets": [ "hostile" ],
+    "effect": "attack",
+    "shape": "blast",
+    "effect_str": "cold_heart_effect",
+    "max_level": 40,
```
Comment (KorGgenT) [inline data/mods/Magiclysm/Spells/attunements/Ice_Elemental.json:63]:
```suggestion
    "max_level": 35,
```

File: data/mods/Magiclysm/Spells/attunements/Magnetism_Mage.json [hunk 6]
```diff
@@ -217,5 +217,33 @@
...
+    "effect": "attack",
+    "shape": "blast",
+    "damage_type": "electric",
+    "min_damage": 20,
+    "max_damage": 120,
+    "damage_increment": 2.5,
+    "min_range": 6,
+    "max_range": 26,
+    "range_increment": 0.5,
+    "max_level": 40,
```
Comment (KorGgenT) [inline data/mods/Magiclysm/Spells/attunements/Magnetism_Mage.json:237]:
```suggestion
    "max_level": 35,
```

**Review by KorGgenT — CHANGES_REQUESTED** [review]
the max level needs to be 35, as stated in the magic balance doc. all of the attunement spells need max level 35 for symmetry reasons.

**Pull request merged**